### PR TITLE
Change landing page ETM to overview > introduction and more

### PR DESCRIPTION
**Pull request by marliekeverweij**
More = 
- I've moved part of the introduction text to the start page and made an English version

Screen Shot 2019-04-29 at 14 20 51

Screen Shot 2019-04-29 at 16 54 42

- Renaming some Sankey-charts to 'easier' names
- Renaming some dashboard items to more understandable names
- Adding chart names for new final demand overview charts

### CODE REVIEW

File: app/assets/javascripts/lib/router.coffee [hunk 1]
```diff
@@ -1,9 +1,24 @@
 class @Router extends Backbone.Router
   routes:
+    "demand/:sidebar(/:slide)" : "demand"
```
Comment (jorisberkhout) [inline app/assets/javascripts/lib/router.coffee]: @marliekeverweij , I don't really get the changes in these lines (3-8). Are you sure they need to be there?

File: app/assets/javascripts/lib/router.coffee [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+    "costs/:sidebar(/:slide)"  : "costs"
+    "overview/:sidebar(/:slide)": "overview"
+    "supply/:sidebar(/:slide)" : "supply"
+    "flexibility/:sidebar(/:slide)" : "flexibility"
+    "data/:sidebar(/:slide)" : "data"
     "report" : "report"
     ":tab/:sidebar(/:slide)" : "load_slides"
     "" : "load_default_slides"
 
+  demand:  (sidebar, slide) => @load_slides('demand', sidebar, slide)
```
Comment (jorisberkhout) [inline app/assets/javascripts/lib/router.coffee]: @marliekeverweij , I don't really get the changes in these lines (13-21). Are you sure they need to be there?

File: app/assets/javascripts/lib/router.coffee (hunk 1 above)
Comment (marliekeverweij) [reply app/assets/javascripts/lib/router.coffee]: You're right! They should not be there. I've deleted them now.

File: app/assets/javascripts/lib/router.coffee (hunk 2 above)
Comment (marliekeverweij) [reply app/assets/javascripts/lib/router.coffee]: You're right! They should not be there. I've deleted them now.

**Comment by jorisberkhout** [comment]
Thanks for the changes @marliekeverweij ! I really like this upgrade to the ETM. Could you merge and deploy this yourself?

**Pull request merged**